### PR TITLE
Implemented CreateContactRequest and CreateContactResponse

### PR DESCRIPTION
**Pull request by srs-binnys**

### CODE REVIEW

File: LiveVox.NET/Models/Contact/Responses/BulkcreateContactResponse.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.Json.Serialization;
+using System.Threading.Tasks;
+using LiveVox.NET.Models.Base;
+
+namespace LiveVox.NET.Models.Contact.Responses
+{
+    public class BulkcreateContactResponse : ILiveVoxResponse
```
Comment (srs-adamr) [inline LiveVox.NET/Models/Contact/Responses/BulkcreateContactResponse.cs]: Typo

File: LiveVox.NET/Models/Contact/Responses/CreateContactResponse.cs [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+namespace LiveVox.NET.Models.Contact.Responses
+{
+    public class CreateContactResponse : ILiveVoxResponse
+    {
+        /// <summary>
+        /// The ID of the newly created contact.
+        /// </summary>
+        [JsonPropertyName("id")]
+        public int Id { get; set; }
```
Comment (srs-adamr) [inline LiveVox.NET/Models/Contact/Responses/CreateContactResponse.cs]: Make long

**Review by srs-adamr — CHANGES_REQUESTED** [review]
Corrections needed

File: LiveVox.NET/Models/Contact/Enumerations/AccountBlock.cs
Comment (srs-adamr) [inline LiveVox.NET/Models/Contact/Enumerations/AccountBlock.cs:1]: This file looks identical to PhoneBlock

**Review by srs-adamr — CHANGES_REQUESTED** [review]

File: LiveVox.NET/Models/Contact/Enumerations/AccountBlock.cs
Comment (srs-binnys) [reply LiveVox.NET/Models/Contact/Enumerations/AccountBlock.cs:1]: Adam, if I merge PhoneBlock and AccountBlock, what can i name it?

Comment (srs-adamr) [reply LiveVox.NET/Models/Contact/Enumerations/AccountBlock.cs:1]: DialBlockOption

Comment (srs-binnys) [reply LiveVox.NET/Models/Contact/Enumerations/AccountBlock.cs:1]: Changes done - new merged file - DialBlockOption

File: LiveVox.NET/Models/Contact/Responses/CreateContactResponse.cs (hunk 2 above)
Comment (srs-binnys) [reply LiveVox.NET/Models/Contact/Responses/CreateContactResponse.cs]: Modified.

File: LiveVox.NET/Models/Contact/Responses/BulkcreateContactResponse.cs (hunk 1 above)
Comment (srs-binnys) [reply LiveVox.NET/Models/Contact/Responses/BulkcreateContactResponse.cs]: Modified.

File: LiveVox.NET/Models/Contact/Responses/BulkcreateContactResponse.cs
Comment (srs-adamr) [inline LiveVox.NET/Models/Contact/Responses/BulkcreateContactResponse.cs:1]: Rename file to match class

Comment (srs-binnys) [reply LiveVox.NET/Models/Contact/Responses/BulkcreateContactResponse.cs:1]: Renamed.

File: LiveVox.NET/Models/Contact/Common/Contacts/BulkUpdateContactDetails.cs
Comment (srs-adamr) [inline LiveVox.NET/Models/Contact/Common/Contacts/BulkUpdateContactDetails.cs:1]: Where is the rest of this object?

File: LiveVox.NET/Models/Contact/Common/Contacts/ExportableField.cs
Comment (srs-adamr) [inline LiveVox.NET/Models/Contact/Common/Contacts/ExportableField.cs:1]: Missing Json tags

File: LiveVox.NET/Models/Contact/Enumerations/ContactField.cs [hunk 3]
```diff
@@ -0,0 +1,467 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.Json.Serialization;
+using System.Threading.Tasks;
+
+namespace LiveVox.NET.Models.Contact.Enumerations
+{
+    [JsonConverter(typeof(JsonStringEnumConverter))]
+    public enum ContactField
```
Comment (srs-adamr) [inline LiveVox.NET/Models/Contact/Enumerations/ContactField.cs]: Consider a better name

File: LiveVox.NET/Models/Contact/Enumerations/CustomFieldEnum.cs [hunk 4]
```diff
@@ -0,0 +1,314 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.Json.Serialization;
+using System.Threading.Tasks;
+
+namespace LiveVox.NET.Models.Contact.Enumerations
+{
+    [JsonConverter(typeof(JsonStringEnumConverter))]
+    public enum CustomFieldEnum
```
Comment (srs-adamr) [inline LiveVox.NET/Models/Contact/Enumerations/CustomFieldEnum.cs]: Consider a better name

File: LiveVox.NET/Models/Contact/Enumerations/ExportField.cs [hunk 5]
```diff
@@ -0,0 +1,659 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.Json.Serialization;
+using System.Threading.Tasks;
+
+namespace LiveVox.NET.Models.Contact.Enumerations
+{
+    [JsonConverter(typeof(JsonStringEnumConverter))]
+    public enum ExportField
```
Comment (srs-adamr) [inline LiveVox.NET/Models/Contact/Enumerations/ExportField.cs]: Consider a better name

File: LiveVox.NET/Models/Contact/Enumerations/SortField.cs [hunk 6]
```diff
@@ -0,0 +1,638 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.Json.Serialization;
+using System.Threading.Tasks;
+
+namespace LiveVox.NET.Models.Contact.Enumerations
+{
+    [JsonConverter(typeof(JsonStringEnumConverter))]
+    public enum SortField
```
Comment (srs-adamr) [inline LiveVox.NET/Models/Contact/Enumerations/SortField.cs]: Consider a better name

File: LiveVox.NET/Models/Contact/Requests/ContactGroups/CreateContactGroupRequest.cs [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+using System.Threading.Tasks;
+using LiveVox.NET.Models.Base;
+using LiveVox.NET.Models.Contact.Common;
+using RestSharp;
+
+namespace LiveVox.NET.Models.Contact.Requests.Contact_Groups
+{
+    public class CreateContactGroupRequest : ILiveVoxRequest
+    {
+        public string? Category { get; set; } = "contact";
```
Comment (srs-adamr) [inline LiveVox.NET/Models/Contact/Requests/ContactGroups/CreateContactGroupRequest.cs:18]: Needs Json Ignore tags, see chat

**Review by srs-adamr — CHANGES_REQUESTED** [review]
Please work on improving enumeration namings, and correct Json Ignore tags for all requests

File: LiveVox.NET/Models/Contact/Common/Contacts/ExportableField.cs
Comment (srs-binnys) [reply LiveVox.NET/Models/Contact/Common/Contacts/ExportableField.cs:1]: This object was not required, hence removed.

File: LiveVox.NET/Models/Contact/Enumerations/ContactField.cs (hunk 3 above)
Comment (srs-binnys) [reply LiveVox.NET/Models/Contact/Enumerations/ContactField.cs]: Renamed it to - SearchConditionContactField

File: LiveVox.NET/Models/Contact/Enumerations/CustomFieldEnum.cs (hunk 4 above)
Comment (srs-binnys) [reply LiveVox.NET/Models/Contact/Enumerations/CustomFieldEnum.cs]: Renamed to ContactDetailCustomField

File: LiveVox.NET/Models/Contact/Enumerations/ExportField.cs (hunk 5 above)
Comment (srs-binnys) [reply LiveVox.NET/Models/Contact/Enumerations/ExportField.cs]: Renamed to ExportableFieldBulkDownload

File: LiveVox.NET/Models/Contact/Enumerations/SortField.cs (hunk 6 above)
Comment (srs-binnys) [reply LiveVox.NET/Models/Contact/Enumerations/SortField.cs]: Renamed to SearchContactsSortField

File: LiveVox.NET/Models/Contact/Requests/ContactGroups/CreateContactGroupRequest.cs (hunk 7 above)
Comment (srs-binnys) [reply LiveVox.NET/Models/Contact/Requests/ContactGroups/CreateContactGroupRequest.cs:18]: Added Json Ignore tags.

**Review by srs-adamr — APPROVED** [review]
Looks good

**Review by srs-vimalk — APPROVED** [review]

**Pull request merged**